packages/ui/src/theme/css/component/expander.scss

```diff
@@ -0,0 +1,131 @@
+.amplify-expander {
+  display: block;
+  background-color: var(--amplify-components-expander-background-color);
+  border-radius: var(--amplify-components-expander-border-radius);
+  box-shadow: var(--amplify-components-expander-box-shadow);
+  width: var(--amplify-components-expander-width);
+}
+
+.amplify-expander__item {
+  display: block;
+  overflow: hidden;
+  margin-top: 1px;
+  box-shadow: var(--amplify-components-expander-item-box-shadow);
+
+  &:first-of-type {
+    margin-top: 0;
+    border-start-start-radius: var(
+      --amplify-components-expander-item-border-start-start-radius
+    );
+    border-start-end-radius: var(
+      --amplify-components-expander-item-border-start-end-radius
+    );
+  }
+
+  &:last-of-type {
+    box-shadow: none;
+    border-end-start-radius: var(
+      --amplify-components-expander-item-border-end-start-radius
+    );
+    border-end-end-radius: var(
+      --amplify-components-expander-item-border-end-end-radius
+    );
+  }
+
+  &:focus-within {
+    position: relative;
+    z-index: 1;
+    box-shadow: var(--amplify-components-expander-item-focus-box-shadow);
```

Comment:
This will make the focus styling over other border styling. It is a really nice trick that I recently learned

Reply:
Nice! Can you please add a brief comment on how this works?